multiqc_clarity/multiqc_clarity.py

```diff
@@ -40,6 +41,11 @@ def __init__(self):
             self.log.debug("No config found for MultiQC_Clarity")
             return None
 
+        if self.schema.get("name_edit_regex"):
+            self.name_edit_regex = self.schema.get("name_edit_regex")
+        else:
+            self.name_edit_regex = None
```

Comment:
I think `get` returns `None` right? So I think this `if`/`else` block is equivalent to just `self.name_edit_regex = self.schema.get("name_edit_regex")`